Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -100,7 +100,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .ordersWithCouponsM4:
             return true
         case .freeTrialSurvey24hAfterFreeTrialSubscribed:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
Nit: Maybe we can remove the feature flag entirely to avoid having to clean it up in the future?

Reply:
Nevermind, I see that we are still keeping the other 24hr-post-subscription notification 😅